boa/src/syntax/parser/statement/iteration/for_statement.rs

```diff
@@ -31,7 +31,7 @@ use crate::{
 ///
 /// [mdn]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for
 /// [spec]: https://tc39.es/ecma262/#sec-for-statement
-#[derive(Debug, Clone, Copy)]
+#[derive(Debug, Clone)]
```

Comment:
Why is this no longer `Copy`?

Reply:
hmm don't know why that was removed, adding back in